src/Analysis/GWASV2/Shared/Covariates/ContinuousCovariates.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+// import './GWASCovariates.css';
+import Covariates from './Covariates';
+import { ACTIONS } from '../constants';
+
+const ContinuousCovariates = ({
+  setMode,
+  // searchTerm,
```

Comment:
enable?

Reply:
![Screenshot 2022-11-21 at 20 22 40](https://user-images.githubusercontent.com/2900303/203141316-1b4b15b2-6a49-447f-8289-833e189ada74.png)

the current app has search. And it is also part of the new design:

![Screenshot 2022-11-21 at 20 23 19](https://user-images.githubusercontent.com/2900303/203141433-8e241a12-a9e0-4157-9344-9ea814995bd3.png)

Reply:
I figured this out and added it back in, however I placed the search bar component within the Covariates.jsx component so that it doesn't render until the Covariates table renders (otherwise you end up with a search bar but nothing shown to search on while the component loads).